reference-architecture/gce-cli/ansible/playbooks/roles/rhel-image/defaults/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+rhel_image_path: '{{ ansible_env.HOME }}/Downloads/rhel-guest-image-7.3-35.x86_64.qcow2'
```

Comment:
Well, this is just a default for variable `rhel_image_path`.. It can be redefined by user altogether. Also, when the user downloads the image with browser, it will be stored in `Downloads` folder by default.